### ISSUE TITLE
[BUG] In tls_mgm module tls_method=TLSv1_2 is not working properly.

### ISSUE THREAD

**Issue by netaskd** [OpenSIPS/opensips#1517]
Hi, I guess a bug found. (opensips 2.4.2)
I try to restrict access only via tls v1.2 on a sip domain. So, doc said:
```
1.8.2. tls_method ([domain]string)
...
TLSv1_2 - means OpenSIPS will accept only TLSv1.2 connections 
...
```
I write a config:
```
## socket
listen=tls:sip.domain.com:5061
## tls
loadmodule "proto_tls.so"
## tls addon
loadmodule "tls_mgm.so"
## options
# default sip.domain.com domain
modparam("tls_mgm", "tls_method", "TLSv1_2")
modparam("tls_mgm", "verify_cert", "0")
modparam("tls_mgm", "require_cert", "1")
modparam("tls_mgm", "ciphers_list", "ALL,!ADH,!LOW,!EXP,!MD5,!RC4,!DES,!3DES,@STRENGTH,!AES256-GCM-SHA384")
modparam("tls_mgm", "certificate","/etc/opensips/tls/cert.pem")
modparam("tls_mgm", "private_key","/etc/opensips/tls/key.pem")
modparam("tls_mgm", "ca_list","/etc/ssl/certs/ca-certificates.crt")
modparam("tls_mgm", "ca_dir", "/etc/ssl/certs")
```
and test via openssl client
```
openssl s_client -no_tls1_1 -no_tls1_2 -connect sip.domain.com:5061
...
Server public key is 2048 bit
Secure Renegotiation IS supported
Compression: NONE
Expansion: NONE
No ALPN negotiated
SSL-Session:
    Protocol  : TLSv1
    Cipher    : ECDHE-RSA-AES256-SHA
...
```
As you can see, TLSv1 allowed, but must be denied. The same with tls1_1.
It would be great to fix it. Thanks

**Comment by razvancrainea** [comment]
Thanks for the report, @netaskd! @rvlad-patrascu managed to reproduce and find the issue.
I've just pushed a fix in c5d6c08. Cheers!

**Comment by rvlad-patrascu** [comment]
Fixed by a67387f
